crates/flox/src/config/mod.rs

```diff
@@ -308,7 +308,7 @@ impl Config {
         let config_file_contents = match fs::read_to_string(&config_file_path) {
             Ok(s) => Ok(Some(s)),
             Err(e) if e.kind() == std::io::ErrorKind::NotFound => {
-                warn!(
+                debug!(
                     "No existing user config file found in {:?}",
```

Comment:
Change this notice to say we are creating a config file?

Reply:
I thought about that, but I think the user doesn't really care much that the file is being created. We don't print anything if the file already exists, so this makes it consistent.

Reply:
no i mean additionally to making it debug.
just when we read a debug log we know its being created and dont have to remember

Reply:
Ah got it. Updated.